### ISSUE TITLE
Convertion IPAddress JSON from IPAddress type

### ISSUE THREAD

**Issue by jPerotto** [bblanchon/ArduinoJson#1845]
I was facing some difficulty removing data placed as a string in JSON, in IPAddress format, such as: 192.168.0.1.

For that I created this function that simply extracts the points, puts the values in a vector, converts to the IPAddress type and returns.

Would it be applicable to ArduinoJson conversions methods?

I implemented it as follows.
```
IPAddress domainIP = stringToIPAddress(jsonConfig["IP"].as<String>());

IPAddress stringToIPAddress(String ip)
{
	uint8_t byteIP[] = {NULL, NULL, NULL, NULL};
	uint8_t ipPosition = NULL;

	for(uint8_t positionString = NULL; positionString < ip.length(); positionString++)
	{
		char byte = ip[positionString];

		if(byte == '.')
		{
			ipPosition++;
			continue;
		}

		byteIP[ipPosition] *= 10;
		byteIP[ipPosition] += byte - '0';
	}

	IPAddress convertIP(byteIP);

	return convertIP;
}
```

I would think it could be used like:

```
domainIP = jsonConfig["IP"].as<IPAddress>();
```

**Comment by bblanchon** [comment]
Hi @jPerotto,

I added the page [How to create converters for IPAddress?](https://arduinojson.org/v6/how-to/create-converters-for-ipaddress/)

TL;DR:

```c++
void convertFromJson(JsonVariantConst src, IPAddress& dst) {
  dst.fromString(src.as<const char*>());
}
```

Best regards,
Benoit
